src/app/components/activity/feed/event-item/event-item.vue

```diff
@@ -107,9 +107,10 @@
 				/>
 
 				<!--
-				This shouldn't ever really show a collapser. It's for the jokers that think it would
-				be fun to make a post with a bunch of new lines.
-			-->
+					This shouldn't ever really show a collapser. It's for the jokers that think it would
+					be fun to make a post with a bunch of new lines.
+					TODO: is item.isLeadOpen wrong?
```

Comment:
figre this out before committing. I don't like TODO lines being left in

Reply:
This is a todo to whoever wrote this, I don't understand where it was expecting to get this value. Can you or nils explain this?

Reply:
I didn't write this, no idea what `isLeadOpen` is supposed to be. It was probably valid at some point and moved to some other part of the code.

Reply:
We can talk about this on chat. It seems like I didn't properly port this one line over to the new "activity view" thing I did. Basically, it's supposed to now call into `isItemLeadOpen` in the `src/app/components/activity/feed/view.ts` class.